compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -447,22 +447,27 @@ execIde telemetry (Debug debug) enableScenarioService options =
             -- performance will be significatly impacted by large log output.
             threshold
             "LanguageServer"
+          damlCacheM <- lookupEnv "DAML_CACHE"
+          damlHomeM <- lookupEnv "DAML_HOME"
+          fallback <- getXdgDirectory XdgCache "daml"
```

Comment:
similar question here, can we deduplicate this with the assistant?

Reply:
now it's only getting DAML_CACHE.